doc/BackupAndRestore.md

```diff
@@ -168,8 +168,16 @@ to restore a backup to that tablet.
 
 As noted in the [Prerequisites](#prerequisites) section, the flag is
 generally enabled all of the time for all of the tablets in a shard.
-If Vitess cannot find a backup in the Backup Storage system, it just
-starts the vttablet as a new tablet.
+By default, if Vitess cannot find a backup in the Backup Storage system,
+the tablet will start up empty. This behavior allows you to bootstrap a new
+shard before any backups exist.
+
+If the `-wait_for_backup_interval` flag is set to a value greater than zero,
```

Comment:
The name "retry" is not accurate here. The current behavior of `-restore_from_backup` is that "no backup exists; starting up empty" is considered a normal, non-error result. It doesn't make sense that adding a retry interval will change this semantic because retries only apply to errors.

Reply:
but you're retrying until success. that's generally what retries are for. this is changing the behavior of restore_from_backup, which only is a single time special case, and happens to startup from empty.

Reply:
> but you're retrying until success

The behavior I'm adding is not "retrying until success". If the backup doesn't exist, "success" is currently defined as "start up empty". There is no need nor expectation to retry anything. The thing you tried (`-restore_from_backup`, which implicitly means "restore from backup if there is one or else start up empty") succeeded.

I need a name that indicates I'm redefining what success means, not just adding a retry until the existing definition of success is met.

> this is changing the behavior of restore_from_backup, which only is a single time special case, and happens to startup from empty.

I don't follow what this means. Can you reword it if I've missed the point?

Reply:
If you look at the new behavior in isolation, the word "retry" makes sense. However, if you take into account the context of how the previous behavior worked -- what users have had to understand since `-restore_from_backup` was first introduced -- then the word "retry" is inaccurate and misleading. In other words, we can't call this "retry" because of legacy.

I'm open to other names that are better than `wait_for_backup`, but not if the name actively misleads users.